pkg/ipfix/testing/mock_ipfix.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Antrea Authors
+// Copyright 2022 Antrea Authors
```

Comment:
Why does it need to be 2022?

Reply:
We don't need to update the old copyright after this file was created.

Reply:
I think this one is updated by `make codegen` automatically.